src/NuGet.Clients/VsExtension/NuGetPackage.cs

```diff
@@ -373,31 +373,33 @@ private NuGet.Configuration.ICredentialService GetCredentialService()
             // Initialize the credential providers.
             var credentialProviders = new List<NuGet.Credentials.ICredentialProvider>();
             
-            TryAddCredentialProvider(
+            TryAddCredentialProviders(
                 credentialProviders,
                 Resources.CredentialProviderFailed_LegacyCredentialProvider,
                 () =>
                 {
                     var packageSourceProvider = new PackageSourceProvider(new SettingsToLegacySettings(Settings));
 
-                    return new CredentialProviderAdapter(new SettingsCredentialProvider(
+                    return new NuGet.Credentials.ICredentialProvider[1] {
```

Comment:
fixed in https://github.com/NuGet/NuGet.Client/pull/664/commits/c076b4b78528f7d65129e2851bd02a446722fb65